frontend/src/metabase/visualizations/echarts/cartesian/scatter/model.ts

```diff
@@ -15,17 +15,20 @@ export function getScatterPlotDataset(
       card,
       data: { rows, cols },
     } = cardSeries;
-    // will be used for breakouts later
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     const columnDescs = cardsColumns[index];
 
     rows.forEach(row => {
       const datum: Record<DataKey, RowValue> = {};
 
       cols.forEach((column, columnIndex) => {
-        datum[getDatasetKey(column, card.id)] = row[columnIndex];
-
-        // TODO handle breakouts
+        const rowValue = row[columnIndex];
+        datum[getDatasetKey(column, card.id)] = rowValue;
```

Comment:
Just made it so that it will only include the breakout key, however for the dimension it will use the non-breakout key since the key in `chartModel.dimension.dataKey` does not include the breakout